cves/CVE-2013-6660.yml

```
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-200
```

Not sure if this change is strictly necessary, but Prof. Meneely's example followed this format.
```suggestion
CWE: 200
```